addons/binding/org.openhab.binding.yeelight/README.md

```diff
@@ -0,0 +1,79 @@
+# Yeelight Binding
+
+This binding integrates the [Yeelight Lighting Product](https://www.yeelight.com/).
+
+
+## Supported Things
+
+[Yeelight LED White Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-w)
+
+[Yeelight LED Color Bulb](https://www.yeelight.com/zh_CN/product/wifi-led-c)
+
+[Yeelight LED Color Stripe](https://www.yeelight.com/zh_CN/product/pitaya)
+
+Project will be updated to support future product.
+
+
+## Preconditions
+
+To control Yeelight device with this binding, you need to connect the device to your
+local network at first with Yeelight app, the app could be downloaded from apple store
+or google play.
+
+Then you need to activate LAN control mode by enable Developer Mode in device settings, after which it will become discoverable by openHAB.
+
+## Discovery
+
+As SSDP defined, there are actually two kinds of discover message: searching and
```

Comment:
As the bulbs support SSDP, I would have very much expected that the binding implements a UPnPDiscoveryParticipant. Why isn't that the case?

Reply:
Since I didn't write most of the code I am not sure.